test-suite/misc/universes/dune

```diff
@@ -1,8 +1,9 @@
 (rule
  (targets all_stdlib.v)
  (deps
+   build_all_stdlib.sh
```

Comment:
Oh good catch! Actually the problem was introduced in PR #8421, when you use `(run foo)` , `foo` is added to the dependencies, but not after we switched to `bash`

Another way to deal with this is to use:
```
(bash "%{dep:./build_all_stdlib.sh}")
```

Reply:
I've verified #8421 and that seems to be the only problem of this kind.